modules/index.js

```diff
@@ -1,11 +1,12 @@
-const danbooru  = require('./danbooru')
-const flickr    = require('./flickr')
-const million   = require('./million')
-const ping      = require('./ping')
-const quote     = require('./quote')
-const random    = require('./random')
-const urban     = require('./urban')
-const troll     = require('./troll')
+var danbooru  = require('./danbooru')
+var flickr    = require('./flickr')
+var million   = require('./million')
+var ping      = require('./ping')
+var quote     = require('./quote')
+var random    = require('./random')
+var troll     = require('./troll')
+var steam     = require('./steam')
+var urban     = require('./urban')
```

Comment:
Let's not use `var`. `const` is ES6, which is what this repo now uses. You might need to update your node version to latest if you're getting errors.